apps/web/src/components/ScreeningDecisions/ScreeningDecisionDialog.tsx

```diff
@@ -384,6 +384,42 @@ export const ScreeningDecisionDialog = ({
     }
   };
 
+  const triggerLabel = (): React.ReactNode => {
+    if (!hasBeenAssessed) {
+      return poolSkill?.type === PoolSkillType.Nonessential
+        ? intl.formatMessage(poolCandidateMessages.unclaimed)
+        : intl.formatMessage(poolCandidateMessages.toAssess);
+    }
+
+    return initialValues?.assessmentDecision === "noDecision" ? (
+      <>{intl.formatMessage(commonMessages.notSure)}</>
+    ) : (
+      <>
+        {intl.formatMessage(
+          initialValues?.assessmentDecision
+            ? getTableAssessmentDecision(initialValues.assessmentDecision)
+            : commonMessages.notFound,
+        )}
+        {initialValues?.assessmentDecision === AssessmentDecision.Successful &&
+        !educationRequirement ? (
+          <span
+            data-h2-color="base(gray.darker)"
+            data-h2-text-decoration="base(none)"
+            data-h2-display="base(block)"
+          >
+            {intl.formatMessage(
+              initialValues?.assessmentDecisionLevel
+                ? getAssessmentDecisionLevel(
+                    initialValues.assessmentDecisionLevel,
+                  )
+                : commonMessages.notFound,
+            )}
+          </span>
+        ) : null}
+      </>
+    );
+  };
+
```

Comment:
Can we move this out to it's own component. We [shouldn't be defining components inside of other components](https://react.dev/learn/your-first-component).